src/app/website/gatsby-config.js

```diff
@@ -37,7 +37,10 @@ s'éclairer directement sur les pages du web qu’ils visitent**.`,
       resolve: `gatsby-source-filesystem`,
       options: {
         name: `contents`,
-        path: `${__dirname}/src/contents`
+        path:
+          process.env.FACET === 'DisMoi'
```

Comment:
Check line 6 for correct spelling … 

Or better check `process.env.FACET === 'lmel'` so that `contentsLeMeme` is specific to `lmel` …